low_sell_products_report/wizards/low_sell_products_report.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import datetime
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import fields, models
+
+
+class LowSellProductsReportWizard(models.TransientModel):
+    _name = "low_sell_products_report_wizard"
+    _description = "Wizard to query low sell products"
+
+    product_type = fields.Selection(
+        selection=[
+            ("product_product", "By product variants"),
+            ("product_template", "By product templates"),
+        ],
+        default=lambda s: s.env.company.low_sales_report_default_product_type,
+        required=True,
+    )
+
+    date_start = fields.Date(
+        default=lambda self: (
+            datetime.date.today()
+            - relativedelta(
+                months=int(self.env.company.low_sales_report_default_period)
+            )
+        ),
+        required=True,
+    )
+    date_end = fields.Date(default=datetime.date.today(), required=True)
+
+    sold_quantity_limit = fields.Float(
```

Comment:
Shouldn't you limit this float to the number of digits for product quantity used in the system?

Reply:
I dont see where the number of digits for the product quantity is declared

https://github.com/odoo/odoo/blob/06f18f44507553bfdd292a8a36762e2d3ad93a30/addons/sale/models/sale_order_line.py#L116